.github/workflows/label-all.yml

```diff
@@ -0,0 +1,14 @@
+name: "Issue and PR Labeler"
+on:
+  pull_request_target:
```

Comment:
This is the first time I'm seeing us use `pull_request_target`. Is there a reason we can't use `pull_request` here instead?

Reply:
yeah I just copied this from the labeler one, but you are right. `pull_request` would also be fine.